modules/data-entry/src/main/frontend/src/questionnaire/Form.jsx

```diff
@@ -546,7 +561,7 @@ function Form (props) {
             : ""
           }
           {
-            data && data['jcr:isCheckedOut'] ?
+            wasCheckedOut ?
```

Comment:
Out of scope, but why are we displaying "Another user is editing" even if we are in view mode?

Suggestion:
```suggestion
            isEdit && wasCheckedOut ?
```

Reply:
I was surprised too, but this was the behavior before the refactoring. I'm OK with the change. The information is useful though, I can see someone else is editing before starting to edit.

Reply:
So, do we apply this suggestion or not?

Reply:
> The information is useful though, I can see someone else is editing before starting to edit.

You said the info is useful. Also it's out of scope, let's not alter the behavior.